src/Compilers/CSharp/Portable/Symbols/MemberSymbolExtensions.cs

```diff
@@ -418,6 +418,23 @@ internal static bool IsPartialDefinition(this Symbol member)
             return sms?.IsPartialDefinition == true;
         }
 
+        internal static bool ContainsTupleNames(this Symbol member)
```

Comment:
I think we have a policy of preferring a kind test to a type test whenever possible.